tests/conditional_compilation/conftest.py

```diff
@@ -78,9 +78,10 @@ def pytest_generate_tests(metafunc):
         if "marks" in test:
             extra_args["marks"] = test["marks"]
 
-        params.append(pytest.param(Path(expand_env_vars(model_path)), **extra_args))
-        ids = ids + [model_path]
-    metafunc.parametrize("model", params, ids=ids)
+        test_id = model_path.replace('$', '').replace('{', '').replace('}', '')
```

Comment:
let's use `re.sub` for such replacements

Reply:
replace()  is faster and straight forward